docs/team/mep.md

```diff
@@ -0,0 +1,265 @@
+'Request for MEP', and expects an MEP issue from the contributor. If there is no further action from
+the contributor for a period of time, the PR/issue is considered as stale and it may get assigned to
+an MMTk team member or may be closed.
+
+Note that a team member may escalate an PR/issue for normal discussion, rather than requesting for
+MEP. As we discussed, MEP is a heavy-weight process, and we should not abuse requesting it.
+
+## Review an MEP
+
+### Criteria
+
+The MMTk team will first decide if the proposal meets the criteria for being an MEP. If it does not
+meet the criteria, the proposal issue will be closed, and related changes should be treated as a
+normal PR/issue.
+
+The criteria for what need to be an MEP is mostly subjective, based on a consensus model within the
+MMTk team. We also provide a list of exemption for what do not need to be an MEP.
+
+#### MEP Exemption
+
+MEP is intended to help avoid design changes that may have profound negative impact in the future.
+Some changes will not have profound impact, and can be easily reverted if necessary. They should be
+exempt from the heavy-weight MEP process, and should not be escalated to request for MEP. The
+exemption is intended to ensure that we won't abuse using MEP and that we won't impose burden on the
+contributors to submit an extra MEP proposal. An exempted PR may still be escalated for team
+discussion, but it is exempt from being requested for MEP (submitting a MEP proposal, and going
+through the MEP process).
+
+##### Exemption 1: Well-encapsulated changes
+
+Changes that are well-encapsulated and decoupled intrinsically can be easily corrected in the future
+and will not have profound impact for the future. A PR that has no public API change, and no module
+API change between the top-level modules (`plan`, `policy`, `scheduler`, `util` and `vm` at the time
+of writing) is exempt from MEP.
+
+### Review
+
+The MMTk team will discuss the proposal in weekly meetings. This process may take a while. We will
+keep posting the discussion to the MEP issue, and encourage further inputs from the contributor, and
+the community. An MEP may get updated and refined during the process.
+
+### Outcome
+
+At the end of the review, the MEP will be accepted or rejected based on the consensus of the MMTk
+team. If an MEP is accepted, A PR may follow the MEP and will be reviewed with the normal PR review
+process.
+
+If an MEP is rejected, future related MEPs may not be reviewed again unless they are substantially
+different. We encourage people to get involved in the review discussion, and refine the proposal so
+it will be accepted.
+
+## Communication
+
+After an MEP is implemented, the MMTk team shall announce the significant changes as the results of
+the MEP to make them known to the community.  We shall communicate with our sponsors about such
+changes, too, in our regular meetings.
+
+<!--
```

Comment:
This should be omitted.

Reply:
This helps vim keep the same indentation and text width so that it can be automatically formatted (with the command `gqap` in vim).  Anyone who edit this document will have the same setting.

Generally speaking, letting text files wrap lines at a specific width makes it more friendly to readers who read the file locally with a text editor (Vim, VSCode, etc.).